app/models/report.rb

```diff
@@ -6,32 +6,34 @@
 
 class Report
   include Blacklight::Configurable
-  include CsvConcern
   include DateFacetConfigurations
   include Blacklight::Searchable
 
   class << self
     include ValueHelper
   end
 
+  COLUMN_SELECTOR_COLUMN_SIZE = 3
+  ROWS_PER_PAGE = 100
+
   REPORT_FIELDS = [
     {
       field: :druid, label: "Druid",
       proc: ->(doc) { doc.druid },
-      sort: true, default: true, width: 100, download_default: true
+      sort: true, default: true, width: 100
```

Comment:
We no longer need to distinguish between whether to make a field downloadable by default and whether to make a field visible by default. These concerns are now aligned. We only made this change as a concession when discovering how broken our outdated client-side library had become, I believe.